### PR TITLE
Optional proxy to avoid issues while logging in

### PR DESCRIPTION
**Pull request by Pietrodjaowjao**
dont merge yet, i will wait for moe to send the proxy address first

### CODE REVIEW

File: main.cs [hunk 1]
```diff
@@ -448,12 +449,22 @@ public override void Load()
         GodMode = Config.Bind("Client Options", "GodMode", false);
         AutoRehost = Config.Bind("Client Options", "AutoRehost", false);
 
+        //Proxy
+        UseProxy = Config.Bind("Client Options", "UseProxy", false);
```
Comment (Tommy-XL) [inline main.cs:453]: Client options also need add in `ClientOptionsPatch`

Comment (Pietrodjaowjao) [reply main.cs:453]: ik, but right now im using only for the .cfg file

Comment (Pietrodjaowjao) [reply main.cs:453]: later i will add it

**Comment by Pietrodjaowjao** [comment]
chat is this correct

**Comment by Pietrodjaowjao** [comment]
it works but this proxy does not exist 💀

**Comment by Pietrodjaowjao** [comment]
so like

**Comment by Pietrodjaowjao** [comment]
image

**Comment by Pietrodjaowjao** [comment]
but when the proxy exists, it works just fine 😎 
image

**Comment by Pietrodjaowjao** [comment]
i might have fixed the issue with another fix.

**Pull request closed without merging**